Dictionary/DICT_THE_BIG_BANG_THEORY.txt

```diff
@@ -0,0 +1,78 @@
+THE BIG BANG THEORY

```

Comment:
We already have a dictionary for the big bang theory. If there are any characters missing from that file, please add them to it.